libs/components/modals/testing/src/modules/confirm/confirm-harness.ts

```diff
@@ -61,7 +60,24 @@ export class SkyConfirmHarness extends SkyComponentHarness {
   }
 
   /**
-   * Gets the confirm component's custom buttons.
+   * Gets a specific confirm custom button that meets certain criteria.
+   */
+  public async getCustomButton(
+    filter: SkyConfirmButtonHarnessFilters,
+  ): Promise<SkyConfirmButtonHarness> {
+    const confirmType = await this.getType();
+
+    if (confirmType !== SkyConfirmType.Custom) {
+      throw new Error(
+        'Cannot get custom button non `Custom` type confirm modals.',
```

Comment:
I'm guessing this should be 'Cannot get custom buttons for non-custom confirm type.'